log10/feedback/feedback_task.py

```diff
@@ -81,10 +81,18 @@ def get(self, id: str) -> httpx.Response:
 @click.option("--name", prompt="Enter feedback task name", help="Name of the task")
 @click.option("--task_schema", prompt="Enter feedback task schema", help="Task schema")
 @click.option("--instruction", help="Task instruction", default="")
-def create_feedback_task(name, task_schema, instruction):
+@click.option(
+    "--completion_tags_selector",
```

Comment:
I'm thinking about the naming. how about just `--completion_tags`? does this filter the completions with the tags and create the inbox for a certain task? Maybe explain a bit more or tell what this does in the `help` string.

Reply:
It is a selector, so for consistency with the api naming - think we should include the "selector" part.
And we may add tags to tasks and feedback themselves, so also to disambiguate from that